### ISSUE TITLE
Entering last prereg phase closes registration

### ISSUE THREAD

**Issue by PiQuer** [inbaz/ers#749]
To reproduce: on the ERS test server shift the prereg phase 1 and prereg phase 2 to dates in the past, then go to the product page, where it sais "PreReg closed".

**Comment by inbaz** [comment]
That's correct, this is the possibility to close prereg. What's the expected behaviour? When open up issues the easiest way to understand what's the problem is to answere these three questions: https://github.com/inbaz/ers/wiki/How-to-create-an-issue

**Comment by PiQuer** [comment]
Let's take for example the ticket "Full event + Gala show ticket", no agegroup. Three prices are configured corresponding to the following deadlines:

- "Prereg phase 1": 31.03.2016, 127,50 €
- "Prereg phase 2": 01.07.2016, 150,00 €
- "no deadline": 170 €

I would expect that on 02.07.2016 the preregistration is still open, and a price of 150 € is displayed. Instead, the preregistration is closed.

To reproduce, set the deadline "Prereg phase 1" to 11.03.2016 and "Prereg phase 2" to 12.03.2016 (assuming this simulates entering "Prereg phase 2" without changing the system time), then go to the Frontend->Products page to find the prereg closed.
